File: leetcode-submissions/3223_Minimum_Length_of_String_After_Operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        vector<int>freq(26,0);
        for(int i=0;i<s.length();i++){
                freq[s[i]-'a']+=1;
            if(freq[s[i]-'a']==3){
                freq[s[i]-'a']-=2;
                
            }
            
        }
        int result=0;
        for(int i=0;i<26;i++){
            if(freq[i]==0){
                continue;
            }
                result+=freq[i];
        }
        return result;
        
    }
};

File: leetcode-submissions/3223_Minimum_Length_of_String_After_Operations.ts
function minimumLength(s: string): number {
    const freq:number[]=new Array(26).fill(0);
    for(const c of s){
        if(c>='a' && c<='z'){
            let index:number =c.charCodeAt(0)-'a'.charCodeAt(0);
            freq[index]+=1;
            if(freq[index]>2){
                freq[index]-=2;
            }
        }
    }
    let ans :number =0;
    for(const f of freq){
if(f===0) continue;
ans+=f;
    }
    return ans ;
};